bindings/pyroot_experimental/PyROOT/python/JupyROOT/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
```

Comment:
The `ROOT` python package does not have a `CMakeLists.txt` for itself, everything is handled in `pyroot_experimental/PyROOT/CMakeLists.txt`. I would do the same (at least for now) and move this content into the PyROOT CMakeLists.txt.